gpio_controllers/src/gpio_command_controller.cpp

```diff
@@ -394,8 +405,18 @@ void GpioCommandController::apply_state_value(
     state_msg.interface_values[gpio_index].interface_names[interface_index];
   try
   {
+    auto state_msg_interface_value_op =
+      state_interfaces_map_.at(interface_name).get().get_optional();
+
+    if (!state_msg_interface_value_op.has_value())
+    {
+      RCLCPP_DEBUG(
+        get_node()->get_logger(), "Unable to retrieve the data from state: %s \n",
+        interface_name.c_str());
+    }
+
```

Comment:
```suggestion
    }
    else
    {
```

here you will continue and access .value(). Here you will need a else statement